### PR TITLE
Add sarah's results

### PR DESCRIPTION
**Pull request by sarahlc888**
- Final val loss: 3.980113040585702 (varies slightly from the plot, which considers randomly sampled eval batches rather than the full eval set)
- https://api.wandb.ai/links/sachen/n5faddjv
- Added SWiGLU activation fn, used parallel ffn and attn, used torch.compile.
- I also experimented with adding more layers and using a longer context window, but without thorough hyperparameter tuning, they did not yield further gains

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks! too bad the other architectural changes didn't result in better performance.
